src/styles/vars.scss

```diff
@@ -0,0 +1,10 @@
+$cell-width: 100px;
+$cal-padding: 10px;
+$cal-gap: 1px;
+$calendar-width: $cell-width * 7 + $cal-padding + $cal-gap * 6;
+
+$cell-color: #eee;
+
+$cal-days: "mon" 1, "tue" 2, "wed" 3, "thu" 4, "fri" 5, "sat" 6, "sun" 7;
```

Comment:
Probably should start from 0, we don't need any margin when month starts on `mon`